src/main/java/loci/common/DateTools.java

```diff
@@ -281,7 +282,20 @@ protected static Instant parseDate(String date, String format, String separator)
         timestamp = timestamp.plus(ms);
       }
       catch (IllegalArgumentException e) {
-        LOGGER.debug("Invalid timestamp '{}'", date);
+        LOGGER.debug("Invalid timestamp '{}' for current locale", date);
+        // if the date couldn't be parsed with the current locale,
+        // try to parse with an English-language locale
+        // this is helpful for dates that use a three-letter month
+        DateTimeFormatter usParser = parser.withLocale(Locale.US);
```

Comment:
As a potential RFE, it would be interesting to skip the fallback if the locale is already English-based or the date pattern is locale-agnostic.